<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Livedoc_Widget</class>
 <widget class="QMainWindow" name="Livedoc_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Livedoc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QWidget" name="noticeWidget" native="true">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>179</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>179</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>179</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>179</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="noticeLayout">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="lblChangeTo">
         <property name="text">
          <string>Do you want to navigate to .....?</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnChange">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNoChange">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Livedoc_Browser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionAutoNavigate"/>
  </widget>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="go-previous" resource="../res/res_core.qrc">
     <normaloff>:/core/go-previous.png</normaloff>:/core/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next" resource="../res/res_core.qrc">
     <normaloff>:/core/go-next.png</normaloff>:/core/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="go-home" resource="../res/res_core.qrc">
     <normaloff>:/core/go-home-10.png</normaloff>:/core/go-home-10.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionAutoNavigate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-import" resource="../res/res_core.qrc">
     <normaloff>:/core/document-import-2.png</normaloff>:/core/document-import-2.png</iconset>
   </property>
   <property name="text">
    <string>Auto Navigate</string>
   </property>
   <property name="toolTip">
    <string>Toggle Auto Navigate</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Livedoc_Browser</class>
   <extends>QTextBrowser</extends>
   <header>gui/livedoc_browser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/res_core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>textBrowser</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>actionPrevious</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textBrowser</sender>
   <signal>forwardAvailable(bool)</signal>
   <receiver>actionNext</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>textBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevious</sender>
   <signal>triggered()</signal>
   <receiver>textBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHome</sender>
   <signal>triggered()</signal>
   <receiver>textBrowser</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
